css/main-styles.css

```diff
@@ -0,0 +1,1043 @@
+}
+.contact-detail-title p {
+  margin-bottom: 20px;
+  font-size: var(--texto);
+  font-weight: 500;
+  color: var(--grey);
+}
+.contact-detail-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+}
+.contact-form-box {
+  width: 80%;
+  margin-bottom: 20px;
+}
+.form-message {
+  display: inline;
+  width: 80%;
+  margin-bottom: 20px;
+}
+.form__info {
+  width: 100%;
+  height: 30px;
+  padding-left: 10px;
+  margin-bottom: 10px;
+  background-color: var(--just-white);
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+}
+.form__text {
+  width: 100%;
+  height: 150px;
+  padding-left: 10px;
+  margin-bottom: 10px;
+  background-color: var(--just-white);
+  border: 1px solid rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+}
+.button-box {
+  margin-left: 10px;
+  padding-left: 50px;
+}
+.contact-button {
+  width: 80%;
+  margin: 0 auto;
+  height: 40px;
+  background-color: var(--blue-soft);
+  border: none;
+  border-radius: 4px;
+  color: var(--just-white);
+  font-weight: 700;
+  font-family: "et-line", sans-serif;
+}
+footer {
+  position: relative;
+  width: 100%;
+  min-width: 320px;
+  height: 100px;
+  padding-top: 30px;
+  background-color: var(--just-white);
+}
+.main-footer-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.footer-copy-box {
+  margin: 10px 0;
+}
+.footer-logo-box a {
+  padding-right: 2.5px;
+}
+.icon-facebook {
+  font-size: var(--iconos);
+  font-weight: 700;
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+.icon-twitter {
+  font-size: var(--iconos);
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+.icon-dribbble {
+  font-size: var(--iconos);
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+.icon-tumblr {
+  font-size: var(--iconos);
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+.icon-linkedin {
+  font-size: var(--iconos);
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+.icon-googleplus {
+  font-size: var(--iconos);
+  background-color: var(--black);
+  color: var(--just-white);
+  border-radius: 50%;
+}
+@keyframes hero {
+  0% {
+    background-image: url(../Images/home-bg-2.jpg);
+  }
+  25% {
+    background-image: url(../Images/home-bg-2.jpg);
+  }
+  30% {
+    background-image: url(../Images/home-bg.jpg);
+  }
+  60% {
+    background-image: url(../Images/home-bg.jpg);
+  }
+  75% {
+    background-image: url(../Images/home-bg-2.jpg);
+  }
+  100% {
+    background-image: url(../Images/home-bg-2.jpg);
+  }
+}
+@media only screen and (max-width: 576px) {
+  .navbar-menu {
+    position: fixed;
+    left: -100%;
+    top: 5rem;
+    flex-direction: column;
+    background-color: var(--just-white);
+    width: 100%;
+    border-radius: 10px;
+    text-align: center;
+    transition: 0.3s;
```

Comment:
when defining a transition be specific, don't leave the timing out, and make it so it only applies to the specific attribute